pkg/services/ngalert/schedule/compat.go

```diff
@@ -40,28 +41,28 @@ func stateToPostableAlert(alertState *state.State, appURL *url.URL) *models.Post
 	// encode the values as JSON where it will be expanded later
 	if len(alertState.Values) > 0 {
 		if b, err := json.Marshal(alertState.Values); err == nil {
-			nA[ngModels.ValuesAnnotation] = string(b)
+			nA[alertingModels.ValuesAnnotation] = string(b)
 		}
 	}
 
 	if alertState.LastEvaluationString != "" {
-		nA[ngModels.ValueStringAnnotation] = alertState.LastEvaluationString
+		nA[alertingModels.ValueStringAnnotation] = alertState.LastEvaluationString
 	}
 
 	if alertState.Image != nil {
-		nA[ngModels.ImageTokenAnnotation] = alertState.Image.Token
+		nA[alertingModels.ImageTokenAnnotation] = alertState.Image.Token
 	}
 
 	if alertState.StateReason != "" {
-		nA[ngModels.StateReasonAnnotation] = alertState.StateReason
+		nA[alertingModels.StateReasonAnnotation] = alertState.StateReason
 	}
 
 	if alertState.OrgID != 0 {
-		nA[ngModels.OrgIDAnnotation] = strconv.FormatInt(alertState.OrgID, 10)
+		nA[alertingModels.OrgIDAnnotation] = strconv.FormatInt(alertState.OrgID, 10)
```

Comment:
OrgIDAnnotation is still defined in `alert_rule.go`, do we want to completely replace it too?

Reply:
Same for `RuleUIDLabel` below, that way we get rid of even more consts?